src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -241,6 +242,29 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
       .length,
   )
 
+  const isOpportunityMature = useMemo(() => {
+    if (!isRunePool) return true
+    if (!isRunePoolOpportunity(earnOpportunityData)) return false
```

Comment:
nitpick: prefer undefined over `false` since this only applies to RUNEPool and regular savers shouldn't be concerned about this logic

```suggestion
    if (!isRunePoolOpportunity(earnOpportunityData)) return
```

Reply:
Fixed in [4abc315](https://github.com/shapeshift/web/pull/7472/commits/4abc3154365f8295c06c2fb383b10d762be64282)

Reply:
Keeping comment open, see inline comment at consumption time